### ISSUE TITLE
Browser API and HTML5 API Interaction Support

### ISSUE THREAD

**Issue by nazarepiedady** [pyscript/pyscript#135]
> Python with JavaScript: Bi-directional communication between Python and Javascript objects and namespaces

Does the sentence above mean that in the future will be possible to interact with the Web Browser APIs and HTML5 APIs using PyScript?

If the answer to the sentence above is negative, do you have the intention to do something like that?

**Comment by Borjagodoy** [comment]
Could be really nice, i go to research how we can help to implement it, could be interesting for the team?

**Comment by wwj718** [comment]
@nazarepiedady 

> Does the sentence above mean that in the future will be possible to interact with the Web Browser APIs and HTML5 APIs using PyScript?

Yes. For example: `from js import document, setInterval, console, navigator`

You can from js to import Browser API and HTML5 API.

[Calling JavaScript functions from Python](https://pyodide.org/en/stable/usage/type-conversions.html?highlight=from%20js%20#calling-javascript-functions-from-python)

**Comment by nazarepiedady** [comment]
> @nazarepiedady
> 
> > Does the sentence above mean that in the future will be possible to interact with the Web Browser APIs and HTML5 APIs using PyScript?
> 
> Yes. For example: `from js import document, setInterval, console, navigator`
> 
> You can from js to import Browser API and HTML5 API.
> 
> [Calling JavaScript functions from Python](https://pyodide.org/en/stable/usage/type-conversions.html?highlight=from%20js%20#calling-javascript-functions-from-python)
> 
> to find what you can import from js
> 
> ```
> import js
> print(dir(js))
> ```
> 
> run the above code in [repl2](https://pyscript.net/examples/repl2.html)
> 
> <img alt="image" width="1418" src="https://user-images.githubusercontent.com/3153878/166650979-6768f405-3ac3-4be9-93f8-15f147dbba82.png">

@wwj718 what is the performance?
